ivy_tests/test_ivy/test_frontends/test_jax/test_nn/test_non_linear_activations.py

```diff
@@ -327,10 +327,10 @@ def test_jax_softmax(
 @handle_frontend_test(
     fn_tree="jax.nn.softplus",
     dtype_and_x=helpers.dtype_and_values(
-        available_dtypes=helpers.get_dtypes("float_and_integer"),
-        large_abs_safety_factor=2,
-        small_abs_safety_factor=2,
-        safety_factor_scale="linear",
+        available_dtypes=helpers.get_dtypes("float_and_complex"),
```

Comment:
```suggestion
        available_dtypes=helpers.get_dtypes("numeric"),
```
`"float_and_complex"` doesn't include integers, so we should use `"numeric"` (or `"valid"`) instead